tests/chainer_tests/functions_tests/rnn_tests/test_function_slstm.py

```diff
@@ -169,6 +170,87 @@ def test_double_backward(self,  backend_config):
         self.check_double_backward(
             self.inputs, self.grad_outputs, self.grad_grad_inputs,
             backend_config)
+"""
+
+
+@testing.parameterize(*testing.product_dict(
+    [
+        {'batch_size': 10, 'n_units': 4},
+        {'batch_size': 8, 'n_units': 5},
+        {'batch_size': 6, 'n_units': 24},
+        {'batch_size': 4, 'n_units': 15},
```

Comment:
These parameterizations seem quite similar but are they testing different scenarios/code paths? I'd understand testing edge cases where values include e.g. 0.

Reply:
I have added tests for different number of dimensions. Let me know if we can include more edge cases.